### ISSUE TITLE
Refactor Log

### ISSUE THREAD

**Issue by kennylavender** [poetapp/node#702]
We are now using a curried log method with pipeP/asyncPipe pretty successfully, I think its time we extract it into a helper or something so that files can import a function that returns log and the types instead of repeating the pattern everywhere.

**Comment by lautarodragan** [comment]
Agreed. This is not needed for mainnet, but we're introducing a bunch of duplicated code which this refactor would avoid.
